[test](mtmv) Fix regression test unstable and add relevant test log #46299 #46546

seawinde opened the pull request:
### What problem does this PR solve?

1 test](mtmv) Fix regression test unstable and add relevant test log
https://github.com/apache/doris/pull/46299
cb6672a7


2 [test](mtmv) Fix sync mv not partition in rewrite test and some other test problems
https://github.com/apache/doris/pull/46546
4bd106b5

3 Fix mv regression test fail occasionally and add some log
https://github.com/apache/doris/pull/47103 
2e614510

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test <!-- At least one of them must be included. -->
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason <!-- Add your reason?  -->

- Behavior changed:
    - [ ] No.
    - [ ] Yes. <!-- Explain the behavior change -->

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. <!-- Add document PR link here. eg: https://github.com/apache/doris-website/pull/1214 -->

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label <!-- Add branch pick label that this PR should merge into -->

seawinde committed:
[test](mtmv) Fix regression test unstable and add relevant test log (#46299)

1. Fix wrong mv regression test case
2. Add relevant test log for location problem

seawinde committed:
[test](mtmv) Fix sync mv not partition in rewrite test and some other test problems (#46546)

Fix sync mv not partition in rewrite because thought mv is already built
wrongly

In the `createMV()` method of `Suite.groovy`, it checks whether the most
recent materialized view of the current database's last table has been
built successfully. If a database has two tables, the synchronization
materialized view of the second table may not be completed, which could
incorrectly lead to the assumption that the build is complete.

To address this issue, modify the test case to ensure that there is at
most one table per database.

hello-stephen commented:
Thank you for your contribution to Apache Doris.
Don't know what should be done next? See [How to process your PR](https://cwiki.apache.org/confluence/display/DORIS/How+to+process+your+PR).

Please clearly describe your PR:
1. What problem was fixed (it's best to include specific error reporting information). How it was fixed.
2. Which behaviors were modified. What was the previous behavior, what is it now, why was it modified, and what possible impacts might there be.
3. What features were added. Why was this function added?
4. Which code was refactored and why was this part of the code refactored?
5. Which functions were optimized and what is the difference before and after the optimization?

seawinde commented:
run buildall

seawinde committed:
[test](mtmv) Fix mv regression test fail occasionally and add some log (#47103)

1. Optimize the test validation logic to check whether the materialized
view is built based on the materialized view name.
2. Wait for the synchronization of the materialized view to be
completed, using the new method.

seawinde commented:
run buildall

hello-stephen committed:
Merge branch 'branch-3.0' into pick_some_pr_to_30_final

hello-stephen commented:
run buildall
